src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1929,8 +1922,6 @@ Move\ panel\ down=Move panel down
 Linked\ files=Linked files
 Group\ view\ mode\ set\ to\ intersection=Group view mode set to intersection
 Group\ view\ mode\ set\ to\ union=Group view mode set to union
-Open\ %0\ URL\ (%1)=Open %0 URL (%1)
-Open\ URL\ (%0)=Open URL (%0)
```

Comment:
Oh, these two were never used?

Reply:
The identifier icon column now collects every url, uri, eprint and doi. On click, the identifier icon column ("linked_id") opens a submenu, like the file icon column. The menuitems of the submenu present the whole link of the collected identifiers (uri etc.) printed out. The removed lines were part of the tooltip, which now presents all the collected links printed out in a very brief form ("URL = ...\nURI = ...\n" etc.).
The icon column is now independet from the field column ("field:url" or "field:doi" etc.) They can print out the stored text of the field in a column.